### DISCUSSION TITLE
Наборы полей по условию при открытии через linkToTable

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#158] [Как это сделать?] [answered]
Можем ли мы настроить так, чтоб при открытии таблицы с помощью linktotable открывался определенный набор полей а не все? И при обычном открытии таблицы отображались все поля.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Это делается через поле префильтра.

Вот вид таблицы при открытии из дерева:

<img width="1180" alt="Снимок экрана 2022-06-23 в 15 58 18" src="https://user-images.githubusercontent.com/55755565/175304775-9b18ab5d-689d-4be0-b6ab-c167d4e5622e.png">

Вот открытое по `linktoTable` с переданным префильтром:

<img width="1185" alt="Снимок экрана 2022-06-23 в 15 58 10" src="https://user-images.githubusercontent.com/55755565/175304882-3333091c-9ea0-40c8-a0ee-95b27e19e4e1.png">

Вот код форматирования открываемой таблицы:

<img width="1040" alt="Снимок экрана 2022-06-23 в 15 57 55" src="https://user-images.githubusercontent.com/55755565/175304951-a6551e45-d650-4b8d-bed9-161687a0aad9.png">

Параметр `fieldhide` управляет видимостью через `глаз`. Это значит, что надо явно задавать скрытие и показ для всех условий.

`forse` означает, что в глазе в скрытом состоянии поле показано не будет.

Это нежесткое скрытие — данные скрытых через `глаз` полей передаются в браузер пользователя и могут быть просмотрены через исходный код страницы.

**Reply by Makhmut2445** [reply]
![image](https://user-images.githubusercontent.com/103171127/175548701-6bbbcab0-6644-4865-8ab5-6b6d00b1e803.png)
Обращаться сохранённому набору полей можем через код-действия, например используя linkToTable?

**Reply by ttm-alexeypolunin** [reply]
Они хранятся в браузере у пользователя, доступа туда никакого нет.
